src/Core/Models/Domain/Login.cs

```diff
@@ -31,7 +33,7 @@ public Login(LoginData obj, bool alreadyEncrypted = false)
         public EncString Totp { get; set; }
         public List<Fido2Credential> Fido2Credentials { get; set; }
 
-        public async Task<LoginView> DecryptAsync(string orgId, SymmetricCryptoKey key = null)
+        public async Task<LoginView> DecryptAsync(string orgId, bool bypassValidation, SymmetricCryptoKey key = null)
```

Comment:
⛏️ Rename to be explicit of what validation are you going to bypass:
```suggestion
        public async Task<LoginView> DecryptAsync(string orgId, bool bypassUriChecksumValidation, SymmetricCryptoKey key = null)
```